lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -12,7 +12,7 @@
 DOCUMENTATION = '''
 ---
 module: docker_swarm_service
-author: "Dario Zanzico (@dariko)"
+author: "Dario Zanzico (@dariko), Jason Witkowski (@jwitko)"
```

Comment:
@dariko Let me know if you have an issue with this.

Reply:
@jwitko No, looks good to me